src/table.py

```diff
@@ -80,11 +81,12 @@
 from dataclasses import dataclass
 from operator import itemgetter
 import weakref
+import pymupdf
 
 # -------------------------------------------------------------------
 # Start of PyMuPDF interface code
 # -------------------------------------------------------------------
-from . import (
+from pymupdf import (
```

Comment:
It probably doesn't matter, but why this change? Did `from . import (...)` stop working for some reason?